requirements.txt

```diff
@@ -1,6 +1,4 @@
-setuptools-markdown
-ConfigArgParse
-mock
-nose
-six
-vcrpy
\ No newline at end of file
+-e .
```

Comment:
Ok, I found this article. https://caremad.io/posts/2013/07/setup-vs-requirement/. Let's go with this approach for now.